### ISSUE TITLE
Too easy to get wrong package with npm

### ISSUE THREAD

**Issue by alflanagan** [dtao/lazy.js#150]
I like the library (so far), and I like the name, but...

`npm search lazy`

finds, among many others,

Name | Description | Author
--------|----------------|----------
lazy|Lazy lists for node|pkrumins  
lazy-js|Lazy instruments for functions|bacher    
**lazy.js**|Like Underscore, but lazier|dtao
lazyjs|Unopinionated Script Loader alternative to CJS and AMD|bevacqua

So I raise this as an issue without, unfortunately, a good suggestion for what to call it instead. There are a *lot* of different packages that start with "lazy", including such obvious ones as lazy-iterator, lazy-functions, lazy-extendable, and (personal favorite) lazy-boy.

**Comment by olsonpm** [comment]
I doubt he'd do a re-branding, though you're right about the problem impacting its visibility.  Neither 'lazy' nor 'js' are google friendly.

**Comment by impinball** [comment]
By the way, Google shows this as the first result for all but the first one. (The first one is a little too generic for Google to properly narrow it down.)

**Comment by dtao** [comment]
Yeah, this one is frustrating for sure. Basically it's my bad. I came up with the name, started the project, and made a lot of progress on it before doing any research into what was already out there with "lazy" in the name. My first choice for npm package was "lazy" but that was taken; then I tried "lazy-js" and "lazyjs" but those were also both taken. "lazy.js" was the only thing left.

If I'd acted sooner, it would've probably been smart to change the name completely to avoid such confusion. However there are now lots of people using lazy and lots of packages depending on it. So I think it's just a dumb reality I now can only apologize for.
